Makefile

```diff
@@ -20,13 +20,13 @@ wasm: ## Generate the WASM for all the contracts
 	$(MAKE) -C ./test-utils $@
 	$(MAKE) -C ./rusk-abi $@
 
-circuits: keys ## Build and test circuit crates
+circuits: ## Build and test circuit crates
```

Comment:
Is this preventing `make circuits` to build properly if the keys are outdated?
Furthermore, this change should not be in a commit prefixed with `transfer-contract: `

Reply:
Yes, it must have been introduced somewhere during the major bump. On the second point: Whoops, you're right. Will fix.